### PR TITLE
Hook up ActionCable for live updating

### PR DESCRIPTION
**Pull request by danielmorrison**
There's a known bug where if you're watching multiple namespaces (/foo,bar) where an update to either will replace the joint state.

### CODE REVIEW

File: app/assets/stylesheets/components/_message.scss [hunk 1]
```diff
@@ -16,6 +16,7 @@
 }
 
 .message {
+//  display: none;
```
Comment (bitmanic) [inline app/assets/stylesheets/components/_message.scss]: This can go, of course.

Comment (danielmorrison) [inline app/assets/stylesheets/components/_message.scss]: Oh right, I went to go add CSS and it all Just Worked™

**Comment by bitmanic** [comment]
@danielmorrison I've got favicons for most of the status combinations. Should we look into updating the favicon via websockets in this branch, too?

**Comment by danielmorrison** [comment]
Sure!

**Pull request merged**